audio/engine.asm

```diff
@@ -2671,13 +2662,11 @@ PlayStereoSFX::
 	ret
 
 LoadChannel:
-; prep channel for use
-; input:
-; 	de:
-	; get pointer to current channel
+; input: de = audio pointer
+; sets bc to current channel pointer
 	call LoadMusicByte
 	inc de
-	and $7 ; bit 0-2 (current channel)
+	and NUM_MUSIC_CHANS ; bit 0-2 (current channel)
```

Comment:
This breaks `make compare`. It should be `maskbits NUM_CHANNELS`. Also, make the same change to the `and $7` in `UpdateChannels`.

Reply:
Why would it break make compare?

Reply:
I didn't know to check make compare. Got it fixed.